src/lib/__snapshots__/create-config.test.ts.snap

```diff
@@ -79,6 +79,10 @@ exports[`should create default config 1`] = `
       "caseInsensitiveInOperators": false,
       "celebrateUnleash": false,
       "changeRequestConflictHandling": false,
+      "createdByUserIdDataMigration": {
+        "enabled": false,
+        "name": "createdByUserIdDataMigration",
+      },
```

Comment:
This looks weird, all `experiments` are `[key: string]: boolean`, but this one breaks that. It doesn't mean it's wrong, but I think we have to be careful here

Reply:
Thank you for pointing that out, it does look a bit strange indeed! I'll take a look at it